README.md

```diff
@@ -54,10 +54,46 @@ Existing middleware include:
 
 - `FunctionName` - dispatch any calls made to a specific Lambda function (by name) to a specified handler class
 - `LogEvent` - log AWS events in a pretty format
-- `SentryCatcher` - catch exceptions and report them to Sentry, including context
-- `SentryMonitoring` - instrument your function code for Sentry's performance monitoring
 - `SetRequestId` - set the `AWS_REQUEST_ID` environment variable for your function code
 
+
+## Additional Middleware
+There are additional middleware available as separate gems that provide specific functionality. They must be explicitly added to your middleware stack in `create_stack`.
+
+### Sentry
+Add the `airfoil-sentry' gem to your Gemfile.This provides three middlewares:
+
+- `SentryCatcher` - catch exceptions and report them to Sentry, including context:
+```ruby
+b.use Airfoil::Middleware::SentryCatcher
+```
+
+- `SentryMonitoring` - instrument your function code for Sentry's performance monitoring (`require "airfoil/middleware/sentry_monitoring"`)
+```ruby
+b.use Airfoil::Middleware::SentryCatcher
+```
+
+- `SentryMonitoring` - instrument your function code for Sentry's performance monitoring (`require "airfoil/middleware/sentry_monitoring"`)
+```ruby
+b.use Airfoil::Middleware::SentryCatcher
+```
+
```

Comment:
Looks like this bit is duplicated. Also, the `SentryCatcher` middleware is show in the code samples both on the `SentryMonitoring` and `Datadog` bits.

Reply:
Oops, good catches. Updated!